yaml/github-actions/security/run-shell-injection.test.yaml

```diff
@@ -83,7 +87,9 @@ jobs:
           echo "PR title did not start with 'octocat'"
           exit 1
           fi
-
+      - name: Print a message
```

Comment:
We need to add a `#ruleid: run-shell-injection` for tests to pass and also to annotate that we expect this to match.